src/core/server/http/http_server.test.ts

```diff
@@ -18,11 +18,7 @@
  */
 
 import { Server } from 'http';
-
-jest.mock('fs', () => ({
-  readFileSync: jest.fn(),
-}));
-
+import { readFileSync } from 'fs';
```

Comment:
The old behavior of this unit test resulted in the creation of an HttpServer with SSL enabled, but an undefined `key` and `certificate`. Technically this worked without errors, but it seemed that this wasn't the original intent of this test. So, I changed this test to use an actual key/cert instead.